### PR TITLE
feat(grid): add container class and tokens

### PR DESCRIPTION
**Pull request by dtsanevmw**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

Adds the container mixin used in boostrap to wrap rows ad columns and additional tokens to control spacings inside

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: projects/angular/src/layout/grid/STYLES.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
 # Grid layout
 
+## CSS Custom Properties
+
+| CSS Custom Property          | Description                                                              |
+| ---------------------------- | ------------------------------------------------------------------------ |
+| --clr-grid-gutter-width      | Grid gutter width                                                        |
+| --clr-grid-container-padding | Grid container padding size. ( Default value is --clr-grid-gutter-width) |
+| --clr-grid-column-padding    | Grid column padding size. ( Default value is --clr-grid-gutter-width)    |
+| --clr-grid-row-margin        | Grid row margin size. ( Default value is --clr-grid-gutter-width)        |
```
Comment (valentin-mladenov) [inline projects/angular/src/layout/grid/STYLES.md:10]: In the `projects/angular/src/utils/variables/_properties.layout.scss` file the default values are actually half of the `--clr-grid-gutter-width`.


```suggestion
| --clr-grid-container-padding | Grid container padding size. ( Default value is half of --clr-grid-gutter-width) |
| --clr-grid-column-padding    | Grid column padding size. ( Default value is half of --clr-grid-gutter-width)    |
| --clr-grid-row-margin        | Grid row margin size. ( Default value is half of --clr-grid-gutter-width)        |
```

File: projects/demo/src/app/grid/grid.demo.scss [hunk 2]
```diff
@@ -8,6 +8,8 @@
 @import '../_utils/clarity-variables';
 
 .clr-example {
+  $border-color: hsl(218, 14%, 30%);
```
Comment (valentin-mladenov) [inline projects/demo/src/app/grid/grid.demo.scss:11]: Can we use token here?

```suggestion
  $border-color: tokens.$cds-alias-object-interaction-border-color;
```

**Review by valentin-mladenov — COMMENTED** [review]
Check my comments

File: projects/angular/src/layout/grid/STYLES.md (hunk 1 above)
Comment (kevinbuhmann) [reply projects/angular/src/layout/grid/STYLES.md:10]: And `--clr-grid-row-margin` is negative.

Comment (dtsanevmw) [reply projects/angular/src/layout/grid/STYLES.md:10]: Thanks for catching it! The values were different and I forgot to change the descriptions after I changed them.

File: projects/demo/src/app/grid/grid.demo.scss (hunk 2 above)
Comment (dtsanevmw) [reply projects/demo/src/app/grid/grid.demo.scss:11]: Yes, totally. Added it.

Comment (dtsanevmw) [reply projects/demo/src/app/grid/grid.demo.scss]: Added in [7079953](https://github.com/vmware-clarity/ng-clarity/pull/1457/commits/7079953691033cf5a38266ce82c7f51be0f19485)

File: projects/angular/src/layout/grid/STYLES.md (hunk 1 above)
Comment (dtsanevmw) [reply projects/angular/src/layout/grid/STYLES.md]: Changed in [7079953](https://github.com/vmware-clarity/ng-clarity/pull/1457/commits/7079953691033cf5a38266ce82c7f51be0f19485)

**Review by valentin-mladenov — APPROVED** [review]
LGTM

**Pull request merged**